### ISSUE TITLE
[feature request] colored media flags & ratings

### ISSUE THREAD

**Issue by zeeyaullah** [jurialmunkey/skin.arctic.zephyr.2#232]
Can we have the colored media flags and movie ratings (imdb, tmdb etc) like in Aura skin please. Is that possible? My request.

**Comment by wsx79** [comment]
Copy folder form aura to zephyr is very simple

**Comment by zeeyaullah** [comment]
Yes, we can but have to repeat the same process after every update. That is the reason why I request it incorporate in the skin itself.
